src/payment/scripts/db-schema.sql

```diff
@@ -1 +1,55 @@
-CREATE TABLE users
+CREATE TABLE tUser (
+	UserID serial PRIMARY KEY,
+	LastName VARCHAR(64) NOT NULL,
+	FirstName VARCHAR(64) NOT NULL,
+	Address VARCHAR(255),
+	City VARCHAR(255),
+	Email VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
Would having this unique cause an issue if a user makes multiple purchases with different credit cards but the same email address?

Reply:
ah - good point then I'll make it a compund unique constraint on email and creditcard